README.md

```diff
@@ -1,14 +1,14 @@
-# v1-series-transformer
+# Alphaville Series Transformer
 
-[![Circle CI](https://circleci.com/gh/Financial-Times/v1-series-transformer/tree/master.png?style=shield)](https://circleci.com/gh/Financial-Times/v1-series-transformer/tree/master)
+[![CircleCI](https://circleci.com/gh/Financial-Times/alphaville-series-transformer.svg?style=svg)](https://circleci.com/gh/Financial-Times/alphaville-series-transformer)
 
-Retrieves Series taxonomy from TME and transforms the series to the internal UP json model.
+Retrieves Alphaville Series taxonomy from TME and transforms the series to the internal UP json model.
 The service exposes endpoints for getting all the series and for getting series by uuid.
 
 # Usage
-`go get -u github.com/Financial-Times/v1-series-transformer`
+`go get -u github.com/Financial-Times/alphaville-series-transformer`
 
-`$GOPATH/bin/v1-series-transformer --port=8080 --base-url="http://localhost:8080/transformers/series/" --tme-base-url="https://tme.ft.com" --tme-username="user" --tme-password="pass" --token="token"`
+`$GOPATH/bin/alphaville-series-transformer --port=8080 --base-url="http://localhost:8080/transformers/series/" --tme-base-url="https://tme.ft.com" --tme-username="user" --tme-password="pass" --token="token"`
```

Comment:
The URLs here and in the endpoints section are not consistent with the code, I think it's expecting a path of /transformers/alphavilleseries.